### ISSUE TITLE
Issue / breaking change with boolean flags with 4.10.0

### ISSUE THREAD

**Issue by ankane** [leejarvis/slop#279]
Hi, thanks for this project! I ran into an issue / breaking change with boolean flags with the latest version (4.10.0).

Script to reproduce:

```ruby
opts = Slop::Options.new
opts.bool "--init"

parser = Slop::Parser.new(opts)
p parser.parse(["--init", "arg1", "arg2"]).to_h
```

With < 4.10.0, this outputs `{:init=>true}`, while with 4.10.0, it outputs `{:init=>false}`.

**Comment by leejarvis** [comment]
Thanks for the report Andrew. Should be fixed in now released [version 4.10.1](https://rubygems.org/gems/slop/versions/4.10.1)

**Comment by ankane** [comment]
Thanks @leejarvis, confirmed that fixes it. Really appreciate the super fast fix and release!
